api/index.js

```diff
@@ -0,0 +1,25 @@
+module.exports = function(app) {
+  app.get('/api/js/global', (req, res) => {
+    res.type('.js');
+    res.send('window.MyAppGlobals = { rootPath: \'/\' };');
+  });
+
+  app.get('/api/css/theme', (req, res) => {
+    res.redirect('/api/themes/default/css/style.css');
+  });
+
+  app.get('/api/src/app/info', (req, res) => {
+    res.json({
+      title: 'Open Metadata Platform'
+    });
+  });
+
+
+  // app.post('/api/test', function(req, res) {
+  //   let user_id = req.body.id;
+  //   let token = req.body.token;
+  //   let geo = req.body.geo;
+
+  //   res.send(user_id + ' ' + token + ' ' + geo);
+  // });
```

Comment:
please remove commented code

Reply:
this was an example on how to get POST params from body request with the `body-parser` npm package, that's why i left it there, i removed it initially then i thought it will be used right away later